SampleBrowser/SampleBrowser/Dependency/DependencyHelper.swift

```diff
@@ -15,9 +15,6 @@ class DependencyHelper {
         let themeManager: ThemeManager = appDelegate.themeManager
         AppContainer.shared.register(service: themeManager)
 
-        let engineProvider = appDelegate.engineProvider
-        AppContainer.shared.register(service: engineProvider)
```

Comment:
I had added the `engineProvider` under the `AppContainer` in the SampleBrowser app. It was a shortcut to easily access it, and now this needs to be better handled with async/await